azure-pipelines.yml

```
@@ -0,0 +1,56 @@
+name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
+variables:
+  CI_NAME: Azure Pipelines
+  CI_BUILD_ID: $(Build.BuildId)
+  CI_BUILD_URL: "https://dev.azure.com/astropy-project/astropy/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp36-* cp37-* cp38-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+
+schedules:
+  - cron: "0 0 * * *"
+    displayName: Daily midnight build
+    branches:
+      include:
+        - master
+    always: false
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: astropy
+    name: OpenAstronomy/azure-pipelines-templates
+    ref: master
+
+trigger:
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - 'v*'
+    exclude:
+    - '*dev*'
+
+jobs:
+- job: noop
+  steps:
+    - checkout: none
+    - bash: echo "Hello World"
+# Only run this job on schedule, to build nightly wheels.
+- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+  - template: publish.yml@OpenAstronomy
+    parameters:
+      # Only upload to the artifacts on master
+      ${{ if eq(variables['Build.SourceBranchName'], 'master') }}:
+        artifact_project : 'astropy'
+        artifact_feed : 'astropy'
+        remove_local_scheme: true
+      test_extras: 'test'
+      test_command: 'pytest -p no:warnings --astropy-header --doctest-rst --pyargs astropy'
```

Why not just let it pick up settings from `setup.cfg`?
```suggestion
      test_command: 'pytest --pyargs astropy'
```

---

Because at this point the tests are being run from a temporary directory that may not have access to setup.cfg. This is basically testing that the wheel installed into a clean environment works properly, so there is no setup.cfg.

---

ohhh ok

---

@Cadair - try removing ``-p no:warnings`` to fix the Windows issue

---

Done